src/main/java/com/box/sdk/BoxTransactionalAPIConnection.java

```diff
@@ -0,0 +1,166 @@
+package com.box.sdk;
+
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+
+import com.eclipsesource.json.JsonObject;
```

Comment:
Yeah gonna stick with the current lib, we can migrate at a later point if it makes sense. I don't think the JSON implementation is exposed externally, so shouldn't be too tough